lib/eventFactory.js

```diff
@@ -426,9 +426,15 @@ class EventFactory extends BaseFactory {
                 .then((workflowGraph) => {
                     modelConfig.workflowGraph = workflowGraph;
 
+                    // create event model
                     return super.create(modelConfig);
                 })
                 .then((event) => {
+                    // Skip creating & starting builds
+                    if (skipMessage) {
+                        return event;
+                    }
+
                     if (modelConfig.type === 'pipeline') {
                         pipeline.lastEventId = event.id;
```

Comment:
is this logic not required if skipping ?

Reply:
You're right. We should still update this field. Will move it